docker-compose.yml

```
@@ -5,7 +5,7 @@ services:
     image: mongo
 
   parse:
-    build: ./backend/
+    # build: ./backend/
```

nah,let's keep that.
```suggestion
    build: ./backend/
```